apps/site/lib/predicted_schedule/schedules.ex

```diff
@@ -0,0 +1,43 @@
+defmodule PredictedSchedule.Schedules do
+  @moduledoc """
+  Provides current predicted schedules
+  """
+
+  alias Site.TransitNearMe
+
+  def get_schedules(route_id, stop_id, opts \\ []) do
```

Comment:
`PredictedSchedules.Schedules.get_schedules` implies that this is somehow only dealing with schedules, and excluding predictions. I would expect a function like this to be `PredictedSchedules.get`.

Reply:
This was existing in this function I moved, I will fix-up these three comments though.